Spokestack/NLUTensorflowTokenizer.swift

```diff
@@ -25,8 +25,8 @@ internal struct BasicTokenizer: Tokenizer {
             })
             // convert the array of characters back to a string
             .joined()
-            //  split the string into alphanumeric word/punctuation components. Also normalize the string again, removing all but alphanumerics and punctuation.
-            .components(separatedBy: NSCharacterSet.alphanumerics.union(NSCharacterSet.punctuationCharacters).inverted)
+            //  split the string into alphanumeric (L* U N*) word/punctuation components. Also normalize the string again, removing all but alphanumerics and punctuation.
+            .components(separatedBy: NSCharacterSet.alphanumerics.subtracting(NSCharacterSet.nonBaseCharacters).union(NSCharacterSet.punctuationCharacters).inverted)
```

Comment:
Hm, OK. I think this works—makes things a bit harder to grok, but the character sets themselves seem right.